GUIDE_SORMAS2SORMAS_CERTIFICATE.md

```diff
@@ -31,18 +33,25 @@ the path to the ``sormas.properties`` file.
 5. After providing the requested data, the certificate files will be generated. <br/>
    The generated certificate has a validity of 3 years. 
    The certificate files will be available in the root SORMAS directory, in the folder ``/sormas2sormas``.
-6. The generated ``.p12`` file should not be shared with third parties. <br/>
+6. A CSV file containing the access data for this instance will also be generated in the folder ``/sormas2sormas``.
+   It will be named ``server-access-data.csv``.
+   The file will contain on the first two columns of the first row the Common Name and the Organization, as provided
+   when creating the certificate. <br/>
+   **Please fill in on the third column the full URL of the server.** <br/>
+   You will also have to set up a user for communicating with other SORMAS instances.
+7. The generated ``.p12`` file should not be shared with third parties. <br/>
    The generated ``.crt`` file will be verified and shared with other SORMAS instances, from which this instance
    will be able to request data. Conversely, in order to enable other SORMAS instances to request data from this 
-   instance, their certificate files should be obtained and added to the local truststore. More details can be found
-   in the next section.
-7. The relevant properties will be automatically set by the script in the ``sormas.properties`` file.
+   instance, their certificate files should be obtained and added to the local truststore. The ``server-access-data.csv``
+   file will also have to be shared so that the access data of this instance is known to other instances. 
+   More details can be found in the next section.
+8. The relevant properties will be automatically set by the script in the ``sormas.properties`` file.
 
 ### Adding a new certificate to the Truststore
 
 To enable other SORMAS instances to request data from this instance, their certificate must be added to the 
-truststore of this instance. After obtaining their certificate file, which should be a ``.crt`` file, please
-follow the next steps:
+truststore of this instance. Furthermore, the access data of other instances must be added to the local server
```

Comment:
As fa as I can see, "request data from this instance" is not supported. The truststore is required for sending and receiving data. The `server-list.csv` is used for sending data to other instances.